SharedSource/UnitTests/Development/NUnit/NUnitExtensionsTests.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+//
+// See the NOTICE file distributed with this work for additional information
+// regarding copyright ownership.  rubicon licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may not use this
+// file except in compliance with the License.  You may obtain a copy of the
+// License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+// License for the specific language governing permissions and limitations
+// under the License.
+//
+
+using System;
+using NUnit.Framework;
+using Remotion.Development.UnitTesting.NUnit;
+namespace Remotion.UnitTests.Development.NUnit
+{
+  [TestFixture]
+  public class NUnitExtensionsTests
+  {
+    [Test]
+    public void AssertThatParameterDoesNotContainWhitespaces_ParameterWithWhitespaces_ThrowsInvalidOperationException ()
+    {
+      var paramName = "param name";
+      Assert.Throws<InvalidOperationException> (() => NUnitExtensions.AssertThatParameterDoesNotContainWhitespaces (paramName));
```

Comment:
Assertion methods private, tests go against the public apis and have to excersise all of them. should wait until we are done kicking out the ones we don't need.